sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py

```diff
@@ -708,8 +708,6 @@ def from_config(
         :keyword file_name: The configuration file name to search for when path is a directory path. Defaults to
             "config.json".
         :paramtype file_name: Optional[str]
-        :keyword cloud: The cloud name to use. Defaults to "AzureCloud".
```

Comment:
why removing this?

Reply:
pylint doesn't allow docstrings to include keywords or parameters that aren't actual arguments for the method/class